File: src/chessgame/pieces.py
# pieces.py

# Define standard starting piece layout for a chess game

def create_starting_pieces():
    pieces = []

    # Add pawns
    for col in range(8):
        pieces.append({"type": "pawn", "color": "white", "pos": [col, 6]})
        pieces.append({"type": "pawn", "color": "black", "pos": [col, 1]})

    # Add major pieces in correct order
    placement = ["rook", "knight", "bishop", "queen", "king", "bishop", "knight", "rook"]
    for col, name in enumerate(placement):
        pieces.append({"type": name, "color": "white", "pos": [col, 7]})
        pieces.append({"type": name, "color": "black", "pos": [col, 0]})

    return pieces

File: next/lib/chess/pieces.js
export const pieceValues = {
  wp: 1,
  bp: -1,
  wn: 3,
  bn: -3,
  wb: 3,
  bb: -3,
  wr: 5,
  br: -5,
  wq: 9,
  bq: -9,
  wk: 100,
  bk: -100,
};

export function getPieceColor(piece) {
  return piece.startsWith("w") ? "white" : "black";
}
